test/src/test/java/jenkins/cli/StopBuildsCommandTest.java

```diff
@@ -182,32 +187,23 @@ private void setupAndAssertTwoBuildsStop(final List<String> inputNames) throws E
         final FreeStyleProject project = createLongRunningProject(TEST_JOB_NAME);
         final FreeStyleProject project2 = createLongRunningProject(TEST_JOB_NAME_2);
 
-        project.scheduleBuild2(0).waitForStart();
-        project2.scheduleBuild2(0).waitForStart();
+        FreeStyleBuild b1 = project.scheduleBuild2(0).waitForStart();
+        j.waitForMessage("Sleeping", b1);
+        FreeStyleBuild b2 = project2.scheduleBuild2(0).waitForStart();
+        j.waitForMessage("Sleeping", b2);
 
         final String stdout = runWith(inputNames).stdout();
 
         assertThat(stdout, equalTo("Build '#1' stopped for job 'jobName'" + LN +
                 "Build '#1' stopped for job 'jobName2'" + LN));
 
-        waitForLastBuildToStop(project);
-        waitForLastBuildToStop(project2);
+        j.assertBuildStatus(Result.ABORTED, j.waitForCompletion(b1));
+        j.assertBuildStatus(Result.ABORTED, j.waitForCompletion(b2));
     }
 
     private FreeStyleProject createLongRunningProject(final String jobName) throws IOException {
         final FreeStyleProject project = j.createFreeStyleProject(jobName);
-        project.getBuildersList().add(createScriptBuilder("sleep 50000"));
+        project.getBuildersList().add(new SleepBuilder(Long.MAX_VALUE));
```

Comment:
Using `SleepBuilder` and `Long.MAX_VALUE` to be consistent with other tests and to more clearly indicate the intention that this is meant to run "forever" before being interrupted.